### PR TITLE
Add test of relevance on calculates

### PR DESCRIPTION
**Pull request by lognaturel**
This is a pattern that we broke in Enketo a while ago. I want to make sure it's explicitly tested here because it is a bit surprising.

### CODE REVIEW

File: src/test/java/org/javarosa/core/model/TriggerableDagTest.java [hunk 1]
```diff
@@ -530,6 +530,33 @@ public void non_relevant_node_values_are_always_null_regardless_of_their_actual_
         assertThat(scenario.answerOf("/data/some-field"), is(intAnswer(42)));
     }
 
+    // Users use relevance on calculates to ensure that calculations are only run when the values
```
Comment (seadowg) [inline src/test/java/org/javarosa/core/model/TriggerableDagTest.java:533]: Is it important to make sure that the calculation isn't executed or just that the calculation isn't "answered"? I'm guessing it's the latter (as that's what this test is about), but the comment suggests it might the former.

Comment (lognaturel) [reply src/test/java/org/javarosa/core/model/TriggerableDagTest.java:533]: Great and deep question. Now I'm no longer 100% sure of the intent. This led me to some strange docs: https://github.com/getodk/docs/pull/1547 I will come back to it.

Comment (lognaturel) [reply src/test/java/org/javarosa/core/model/TriggerableDagTest.java]: I clarified the comment wording to make it clear it's about the node not exposing a value. I've also added tests below to verify that multiplication and addition with empty values result in empty values.

**Comment by lognaturel** [comment]
@latin-panda @garethbowen Could one of you please check to make sure that there are tests in Web Forms around this. It's connected to some questions about empty values that we've had recently.

**Comment by latin-panda** [comment]
@lognaturel I've ported the missing tests to WF (https://github.com/getodk/central-frontend/pull/1916)
